docs/modules/ROOT/partials/snippets/NixOSToken.adoc

```diff
@@ -0,0 +1 @@
+include::partial$generate-token.adoc[]
```

Comment:
I don't see a good reason to create another indirection?

Reply:
This makes the snippet available to the frontend.

Reply:
Because of camelcase naming? So why not rename the original?

Reply:
Not just that, partial/generate-token.adoc is not in the snippet directory.
If you think partial$generate-token.adoc needs to be inlined, we can refactor that separately. I'm merely following the pattern that was already there, to achieve a goal of fixing the frontend.